src/cli/generator/config.rs

```diff
@@ -171,7 +180,7 @@ impl Location<UnResolved> {
 }
 
 impl<A> Headers<A> {
-    pub fn headers(&self) -> &Option<BTreeMap<String, String>> {
+    pub fn get(&self) -> &Option<BTreeMap<String, String>> {
         &self.0
```

Comment:
`self.0` is public. Either remove this implementation or we add a method called `as_btree_map` and make `self.0` private. Option 2 is preferred.

Reply:
done, made `self.0` private